app/assets/stylesheets/application.scss

```diff
@@ -2,8 +2,25 @@
 @import "bootstrap-sprockets";
 @import "bootstrap";
 
+$accent-color: #FF384D;
+$bg-color: #F9F9F9;
+$dark-bg: #464A57;
+$active-dark: #44474B;
+$active-accent: #FF2849;
+
 body {
-  background: #F9F9F9
+  background: $bg-color
+}
+
+body h1, h2, h3, h4, h5, h6, .h1, .h2, .h3, .h4, .h5, .h6 {
+  font-family: serif;
+  font-weight: 500;
+  line-height: 2;
+  color: #353535;
```

Comment:
Color literals like `#353535` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Each selector in a comma sequence should be on its own line